### ISSUE TITLE
I want to contribute, where do I start?

### ISSUE THREAD

**Issue by kof** [cssinjs/jss#1359]
Hey, 

if you want to contribute but don't know where to start, write a comment here, ideally give me some hints on what kind of issue you are looking or:
- level of complexity (low, moderate, high)
- issue type (bug, enhancement, feature)

I will suggest you a few issues and will try to guide you through the requirements.

Best,
Oleg

**Comment by half-halt** [comment]
I am happy to help, I've spent some time messing with concurrent mode, and createUseStyles.

**Comment by kof** [comment]
@half-halt want to send a PR?

**Comment by Issam-AB** [comment]
Yes, that's fine, please email me I want to know more about how contribute good about this.

**Comment by kof** [comment]
@Issam-AB feel free to describe what you want to work on or what you are looking for and what kind of troubles you had so far with contributing!

**Comment by Rishabh-Rathod** [comment]
@kof  _( Not sure if this is best place to put it. )_

I have been working on [css-to-js, a vscode extension](https://github.com/Rishabh-Rathod/css-to-js/)  which can help CssinJs users to migrate from css to CssInJs. 
( which was my need for few of the projects. )

it Handles

- css mediaQueries.
- multiple class or a whole css file conversion in one go.
- css keyframes.

PS - i tried using online tools (like [transform-tools](http://transform.tools/) or popular [css-in-js extention](https://marketplace.visualstudio.com/items?itemName=paulmolluzzo.convert-css-in-js) ) but none of them solved all the issues.

I would be glad if this extension can be of any help to other cssInJs library users.

**Comment by kof** [comment]
@Rishabh-Rathod we have a list of community projects https://cssinjs.org/community-projects?v=v10.5.0 also consider contributing to existing tools if possible

**Comment by Rishabh-Rathod** [comment]
@kof 

Thanks for the list. 
As per the list of projects [jss-codemorphs](https://github.com/codemodsquad/jss-codemorphs) is solving similar problem.
As per my research Pseudo Class Support is missing. 
i have raised an [issue](https://github.com/codemodsquad/jss-codemorphs/issues/12) .. looking forward to contribute to it.

**Comment by jedwards1211** [comment]
It's not that pseudo classes aren't supported, it's just not good about converting multiple references to the same class selector in flat CSS to `$className` references.

**Comment by mdsadiq** [comment]
@kof 
i would like to start my contribution from low complexity and react.
i have just started ts as well
any suggestions ?

**Comment by kof** [comment]
@mdsadiq consider [these](https://github.com/cssinjs/jss/labels/needs%20tests) and [these](https://github.com/cssinjs/jss/issues?q=is%3Aopen+is%3Aissue+label%3Acomplexity%3Alow+) let me know if you need further help in the corresponding issues

**Comment by dksingla** [comment]
Hello @kof,

I would like to solve low complexity bugs. I can work with ts, react.

**Comment by kof** [comment]
hi @dksingla, I have labeled a few https://github.com/cssinjs/jss/issues?q=is%3Aopen+is%3Aissue+label%3Acomplexity%3Alow

Otherwise try to read through the issues and see what will make sense to you
